modules/openapi-generator/src/test/resources/3_0/issue_7533.yaml

```diff
@@ -0,0 +1,17 @@
+openapi: 3.0.1
+info:
+  title: OpenAPI Petstore
+  description: "sample spec"
+  license:
+    name: Apache-2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+servers:
+  - url: /
```

Comment:
Per @jimschubert 
```
Server with no url base specified is meant to be the same server where the spec doc was downloaded from.
```
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#server-object:
```
REQUIRED. A URL to the target host. This URL supports Server Variables and MAY be relative, to indicate that the host location is relative to the location where the OpenAPI document is being served. Variable substitutions will be made when a variable is named in {brackets}.
```

Reply:
Hmm rust tests fail because they assume that we keep stripping the / from the end of the server url. Probably better to keep removing it and have future PRs detect the "" use case if they need to. That way this is non-breaking.